limit.go

```diff
@@ -0,0 +1,75 @@
+package pilosa
```

Comment:
we should rename this file to something a little more specific and related to its content.
perhaps `listener.go`. what do you think?